dvc/remote/local/__init__.py

```diff
@@ -75,11 +66,28 @@ def __init__(self, repo, config):
         else:
             self.cache_types = copy(self.DEFAULT_CACHE_TYPES)
 
-        if self.cache_dir is not None and not os.path.exists(self.cache_dir):
-            os.mkdir(self.cache_dir)
+        # A clunky way to detect cache dir
+        # FIXME: why AWS?
```

Comment:
no reason at all, let's resolve or create a ticket for this

Reply:
As far as I can see this is part of some backwards-compatibility code. What do you propose? Creating yet another constant with `'storagepath'` as value?

Reply:
for starters rename the constant to not contain AWS in it, since it's reused for many remotes. I don't have a solution ready for the whole cache detection logic.

Reply:
Done.